.github/workflows/automerge.yaml

```diff
@@ -0,0 +1,63 @@
+name: automerge auto-cherry-picked pr's
+on:
+  pull_request_target:
+    types:
+      - labeled
+      - unlabeled
+      - synchronize
+      - opened
+      - edited
+      - ready_for_review
+      - reopened
+      - unlocked
+    branches-ignore:
+      - master
+  pull_request_review:
+    types:
+      - submitted
+  check_suite:
+    types:
+      - completed
+  status: {}
+
+jobs:
+  automerge:
+    name: Automerge auto-cherry-picked pr
+    runs-on: ubuntu-latest
+    steps:
+      - id: find-prt-comment
+        if: contains(github.event.pull_request.labels.*.name, 'automerge')
```

Comment:
We need only the `automerge` label, it will not be running on the master or parent one you mentioned as it is avoided.

We do not need to copy those labels from parent PR as the author can easily add them on the AutoCherryPicked PRs. It will be one layer or preventive checkup. 

Why do we need an extra label Auto_Merge_Cherrypicked?